cmd/system-probe/modules/ping.go

```diff
@@ -0,0 +1,138 @@
+//go:build linux
+
+package modules
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/api/module"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	sysconfigtypes "github.com/DataDog/datadog-agent/cmd/system-probe/config/types"
+	"github.com/DataDog/datadog-agent/cmd/system-probe/utils"
+	pingcheck "github.com/DataDog/datadog-agent/pkg/networkdevice/pinger"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/gorilla/mux"
+	"go.uber.org/atomic"
+	"google.golang.org/grpc"
+)
+
+const (
+	countParam    = "count"
+	intervalParam = "interval"
+	timeoutParam  = "timeout"
+)
+
+type pinger struct{}
+
+// Pinger is a factory for NDMs Ping module
+var Pinger = module.Factory{
+	Name:             config.PingModule,
+	ConfigNamespaces: []string{},
```

Comment:
What the purpose of namespaces?

Reply:
They are used with the `config` CLI commands to ensure all config namespaces are included in the output. This should contain `"ping"`.